src/g1.rs

```diff
@@ -1,5 +1,7 @@
 //! This module provides an implementation of the $\mathbb{G}_1$ group of BLS12-381.
 
+use core::hash::{Hash, Hasher};
+
```

Comment:
Can you remove this empty line just to keep imports grouped together?

Reply:
ok